source/jormungandr/jormungandr/excluded_zones_manager.py

```diff
@@ -28,15 +28,23 @@
 # www.navitia.io
 
 import boto3
+import pytz
+import shapely.iterops
+from dateutil import parser
 from botocore.client import Config
 import logging
 import json
+import shapely
+import datetime
+from typing import Dict
 
 from jormungandr import app, memory_cache, cache
 from jormungandr.resource_s3_object import ResourceS3Object
 
 
 class ExcludedZonesManager:
+    excluded_shapes = dict()  # type: Dict[str, shapely.geometry]
+
     @staticmethod
     @cache.memoize(app.config[str('CACHE_CONFIGURATION')].get(str('ASGARD_S3_DATA_TIMEOUT'), 24 * 60))
     def get_object(resource_s3_object):
```

Comment:
can you add **_instance_publication_date_**

Reply:
for excluded_zones, they have nothing to do with the data of instance, it does not make sense to relate excluded_zones to instance_publication_date